addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3529,8 +3529,8 @@ const SnippetOptionWidget = Widget.extend({
             return Promise.all(proms);
         });
 
-        const showUI = await this._computeVisibility();
-        this.el.classList.toggle('d-none', !showUI);
+        this.showUI = await this._computeVisibility();
+        this.el.classList.toggle('d-none', !this.showUI);
```

Comment:
Nah I don't think that's a good idea. The most common method is `_computeWidgetVisibility`, which we will likely use for options which only have one widget. 

But about this whole commit purpose: did not it work at some point? I am 99% sure I did not merge the "page options" work with the section showing up all the time :grimacing: